addons/website_event/static/src/scss/event_templates_list.scss

```diff
@@ -97,3 +70,31 @@
         min-height: 200px !important;
     }
 }
+
+.o_wevent_event_date {
```

Comment:
Why is it moved ? Isn't it in the "o_wevent_index" namespace anymore ?

Reply:
Idem as under, we also use it in website_event_sale :/